alexandria/core/tasks.py

```diff
@@ -0,0 +1,9 @@
+from celery import shared_task
+
+
+@shared_task
+def set_content_vector(file_pk):
+    from .models import File
+
+    file = File.objects.get(pk=file_pk)
+    file.set_content_vector()
```

Comment:
Why not move the entire logic here, out of the model? I think it'd be cleaner that way

Reply:
Done